airflow/api_connexion/openapi/v1.yaml

```diff
@@ -2716,6 +2716,15 @@ components:
   # Reusable callbacks
   callbacks: {}
 
+  securitySchemes:
+    BasicAuth:
+      type: http
+      scheme: basic
+    GoogleOpenIdAuth:
```

Comment:
Should we define it as aa OpenID? However, I am not sure. It may be more practical to define it as a JWT if we can support more flow e.g. gcloud.
```
    GoogleOpenId:
      type: openIdConnect
      openIdConnectUrl: https://accounts.google.com/.well-known/openid-configuration
```

Reply:
I think it makes more sense to define it as openIdConnect type, will update it. we can add JWT auth scheme for future bearer based auth backends.